### DISCUSSION TITLE
Dataset run description and metadata should not be fixed on the page

### DISCUSSION THREAD

**Discussion by justinas-kazanavicius** [langfuse/langfuse#3212] [Ideas]
### Describe the feature or potential improvement

Description and metadata are always fixed on the page (they cannot be scrolled out of view), so if the description or metadata are long enough, there won't be enough space to see the dataset run items at all. It is possible to collapse the metadata to give more real estate to the dataset rows, but that is unintuitive and also even after doing that, there might still not be enough space if the screen size is not big enough. For example, on my colleague's laptop with a smaller screen, he could only see a single row even after collapsing the metadata.

**Solution**: dataset run description and metadata elements should not be fixed, so that when scrolling down the page they would not be visible anymore, giving more space to see the dataset run items.

**Comment by marcklingen** [comment]
agree, opened issue to track this: https://github.com/langfuse/langfuse/issues/3213
